Functions/Quantification/xASL_quant_CBF.m

```diff
@@ -292,18 +298,48 @@
 fprintf('%s\n',' model with parameters:');
 
 if x.modules.asl.ApplyQuantification(3)
-    if isfield(x.Q,'LabelingDuration') && isfield(x.Q,'Initial_PLD')
-        switch lower(x.Q.LabelingType)
-            case 'pasl'
-                fprintf('%s',['TI1 = ' xASL_num2str(x.Q.LabelingDuration) ' ms, ']);
-                fprintf('%s',['TI (ms) = ' xASL_num2str(x.Q.Initial_PLD)]);
-            case 'casl'
-                fprintf('%s',['LabelingDuration = ' xASL_num2str(x.Q.LabelingDuration) ' ms, ']);
-                fprintf('%s',['PLD (ms) = ' xASL_num2str(x.Q.Initial_PLD)]);
-        end
-    else
-        fprintf('Labeling duration and initial PLD undefined...\n');
-    end
+	if modules.asl.bMultiPLD
+		% Prepare unique multi-PLD parameters
+		[PLDs, ind] = unique(x.Q.Initial_PLD, 'stable');
```

Comment:
Does `ind` mean `index` or `individual`? Can you write it out?

Reply:
Yes.